autoload/youcompleteme.vim

```diff
@@ -84,6 +84,7 @@ function! youcompleteme#Enable()
     autocmd InsertLeave * call s:OnInsertLeave()
     autocmd InsertEnter * call s:OnInsertEnter()
     autocmd VimLeave * call s:OnVimLeave()
+    autocmd CompleteDone * call s:OnCompleteDone()
```

Comment:
Don't we want to add this autocmd conditionally? When was the CompleteDone event added to Vim?